contracts/src/utils/math/storage.rs

```diff
@@ -6,8 +6,8 @@ pub(crate) trait AddAssignUnchecked<T> {
     fn add_assign_unchecked(&mut self, rhs: T);
 }
 
-impl<'a, const B: usize, const L: usize> AddAssignUnchecked<Uint<B, L>>
-    for StorageGuardMut<'a, StorageUint<B, L>>
+impl<const B: usize, const L: usize> AddAssignUnchecked<Uint<B, L>>
```

Comment:
This is already refactored on `origin/main`

Reply:
yeah, will rebase it. Not a problem